telco-hub/configuration/reference-crs/optional/backup-recovery/policy-backup.yaml

```diff
@@ -0,0 +1,68 @@
+# this policy will  add the backup label to all BMM having infraenvs.agent-install.openshift.io to avoid redeployment when recovery is done
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: bmh-cluster-activation-label
```

Comment:
All the CRs in this file should be placed in a specific namespace, ideally the open-cluster-management-backup ns.